src/components/card/index.tsx

```diff
@@ -1,13 +1,170 @@
 import * as React from 'react'
+import {createNamedContext} from '@reach/utils/context'
+import {useId} from '@reach/auto-id'
+import type * as Polymorphic from '@reach/utils/polymorphic'
+
+const CardContext = createNamedContext<InternalCardContextValue>(
+  'CardContext',
+  {} as InternalCardContextValue,
+)
+
+type InternalCardContextValue = {
+  cardId: string | undefined
+  quiet: boolean
+  horizontal: boolean
+}
+
+type CardProps = {
+  quiet?: boolean
+  horizontal?: boolean
+}
+
+const cardDefaultClasses = `bg-white dark:bg-gray-800 dark:text-gray-200 shadow-sm rounded-lg overflow-hidden sm:p-8 p-5`
+
+const Card = React.forwardRef(function Card(
```

Comment:
I used some patterns in Reach for this component, and it's "overkill" but it also creates a very nice markup structure when it compiles and gives us a lot of forwards options with the card.